### ISSUE TITLE
d912pxy doesn't work with GW2 update 28 July Jormag Rising

### ISSUE THREAD

**Issue by rido-ramadan** [megai2/d912pxy#272]
**Describe the bug**
Crash on start, before character selection screen

**To Reproduce**

1. Run game

**Expected behavior**
Character selection rendered

**Screenshots**

**Standard information**

1. Windows 10 1909
2. i7-8750H, GTX 1070 Max-Q, 16 GB RAM
3. NVIDIA 451.48
4. 2.1.2 - 25th june appveyor build
5. No log

**Additional context**

**Comment by demonpuppet** [comment]
I am having the same issue as well.

**Comment by Kohtas** [comment]
Also having the same issue after the game update.

**Comment by exhale451** [comment]
Same here.

**Comment by Callum17** [comment]
Same issue, d912pxy_v2.1.2_r532 if that helps

**Comment by StonedAstronaut** [comment]
Same issue. Reinstall didn't help.

```
user@msys2 ~ $ tail /c/Games/Guild\ Wars\ 2/d912pxy/log.txt
11C503D [ error ] helper | Exception: E06D7363
11C503D [ error ] helper | Exception: E06D7363
11C503D [ error ] helper | Exception: E06D7363
11C503D [ error ] helper | Exception: E06D7363
11C503D [ error ] helper | Exception: E06D7363
11C503D [ error ] helper | Exception: E06D7363
11C503D [ error ] helper | Exception: E06D7363
11C503D [ error ] GPU command list | throw on "can't close command list" with HR = 0x80070057
11C503D [ note ] note | 3 | "can't close command list" | hr = 80070057
11C503D [ error ] helper | Exception: E06D7363
```

**Comment by megai2** [comment]
game have bad commands in api stream after patch

added a quick fix, should be in latest builds soon

https://ci.appveyor.com/project/megai2/d912pxy/build/artifacts

**Comment by demonpuppet** [comment]
> game have bad commands in api stream after patch
> 
> added a quick fix, should be in latest builds soon
> 
> https://ci.appveyor.com/project/megai2/d912pxy/build/artifacts

When do you expect it to be available?

**Comment by megai2** [comment]
Released all pending changes in v2.2 https://github.com/megai2/d912pxy/releases/tag/v2.2

**Comment by Callum17** [comment]
Impressive turnaround time. For reference I am downloading the unmodified client again - now at 31% on 24Mbps broadband.

Unshamedly sang your praise in the Anet bug report. Hope they have reached out to you. Will try the aforementioned changes once the I have confirmed unmodified client is working.

**Comment by X-TAC** [comment]
2.2 installed, but I am still crashing repeatedly at the second cannon in the first new story instance.  Not sure if that is dx912pxy or something else.  After uninstall, can't get the game to launch.  Is there something else that needs to be repaired after removal?

**Comment by Kohtas** [comment]
> 2.2 installed, but I am still crashing repeatedly at the second cannon in the first new story instance. Not sure if that is dx912pxy or something else. After uninstall, can't get the game to launch. Is there something else that needs to be repaired after removal? (repaired and was able to launch and make it past the the first story instance).

Be sure to uninstall it by running the installer again and selecting the "Remove" option once you get to it, once that is done delete the d912pxy folder as well as the bin folder (just be be safe), then relaunch the game once to repopulate deleted files after that just install d912pxy how you normally would.

**Comment by Kohtas** [comment]
Thank you for the fix, it seems to actually be running much better than last release was for me.

**Comment by X-TAC** [comment]
Did anyone else have problems with the first story instance on 2.2?  Unsure if it is related to dx912pxy or not and would love to re-install it.  Can't go back to DX9 after 7 months of better living.

**Comment by vertikalz** [comment]
https://imgur.com/a/kMWj6eD

Been having this problem since the update, is it related to the workaround? In dialogue boxes, character select, and when I click a character it seems their head textures aren't loading in. Before if I clicked away and back then usually it would be fixed but now it just stays like this.

I tried reinstalling and reloading all shaders and it's still like this. (I replied to one of your reddit comments too but didn't want to start a new thread). GPU is 5700xt
